app/build.gradle.kts

```diff
@@ -55,13 +55,23 @@ android {
     buildTypes {
         debug {
             manifestPlaceholders["targetBitwardenAppId"] = "com.x8bit.bitwarden.dev"
+            buildConfigField(
+                type = "com.bitwarden.authenticatorbridge.manager.model.AuthenticatorBridgeConnectionType",
+                name = "AUTHETNICATOR_BRIDGE_CONNECTION_TYPE",
+                value = "com.bitwarden.authenticatorbridge.manager.model.AuthenticatorBridgeConnectionType.DEV",
+            )
             signingConfig = signingConfigs.getByName("debug")
             isDebuggable = true
             isMinifyEnabled = false
         }
 
         release {
             manifestPlaceholders["targetBitwardenAppId"] = "com.x8bit.bitwarden"
+            buildConfigField(
```

Comment:
Thanks for the inspiration here, @SaintPatrck ! I had forgotten this was possible. I like how this turned out, since have compile time safety for this value.